Renderer/Block/AtomicBlockRenderer.php

```diff
@@ -39,30 +39,29 @@ public function setBlockEntityGuesser(BlockEntityGuesser $blockEntityGuesser)
      */
     public function render(\ArrayIterator &$iterator, array $entities)
     {
+        /** @var ContentBlock $contentBlock */
         $contentBlock = $iterator->current();
         $iterator->next();
 
+        /** @var DraftEntity $entity */
         $entity = $this->getEntity($contentBlock, $entities);
 
-        if (!$entity) {
-            return '';
-        }
-
-        $renderer = $this->blockEntityGuesser->getRenderer($entity);
+        $content = $contentBlock->getText();
 
-        if (!$renderer) {
-            throw new DraftjsException(sprintf('Undefined block entity renderer for type "%s"', strtolower($entity->getType())));
-        }
+        if ($entity) {
+            $renderer = $this->blockEntityGuesser->getRenderer($entity);
 
-        $content = $renderer->render($entity);
+            if (!$renderer) {
+                throw new DraftjsException(sprintf('Undefined block entity renderer for type "%s"', strtolower($entity->getType())));
+            }
 
-        if (empty($content)) {
-            return $content;
+            $content = $renderer->render($entity);
         }
 
         return $this->templating->render('M6WebDraftjsBundle:Block:default.html.twig', [
             'classNames' => $this->buildClassNames($contentBlock),
             'content' => $content,
+            'data' => $contentBlock->getData(),
```

Comment:
Passing data entity is good idea, but we don't use them in the template at moment.
If you do not need it, do not implement it.